src/read_write/ply.rs

```diff
@@ -465,6 +465,9 @@ impl NodeWriter<PointsBatch> for PlyNodeWriter {
     }
 
     fn write(&mut self, p: &PointsBatch) -> io::Result<()> {
+        if p.position.is_empty() {
```

Comment:
How about `if self.point_count == 0 && !p.position.is_empty()` below?

Reply:
I personally prefer the early break, but do not insist on it :)

Reply:
Okay , it's fine like this too.